draft-ietf-quic-multipath.md

```diff
@@ -1346,7 +1346,8 @@ MAX_PATH_ID frames that do not increase the path limit MUST be ignored.
 
 A sender SHOULD send a PATHS_BLOCKED frame (type=0x15228c0d) when
 it wishes to open a path but is unable to do so due to the maximum path identifier
-limit set by its peer;
+limit set by its peer. Note that PATH_BLOCKED frame is informational, as it isn’t
+expected to change the state of peer.
```

Comment:
I think we should be more explicit what we mean here, maybe like:

"Sending a PATH_BLOCKED frame does not imply a particular action from the peer like updating the new Max Path ID value."